python_modules/dagster/dagster_tests/execution_tests/engine_tests/test_step_delegating_executor.py

```diff
@@ -220,6 +216,50 @@ def test_execute_intervals():
     assert TestStepHandler.check_step_health_count >= 3
 
 
+@solid
+def slow_solid(_):
+    time.sleep(2)
+
+
+@pipeline(
+    mode_defs=[
+        ModeDefinition(
+            executor_defs=[test_step_delegating_executor],
```

Comment:
unfortunately still hard to do tests with an in process run worker but non default executor when using Jobs

Reply:
you're right, not sure what I ran in to